README.md

```diff
@@ -53,19 +60,15 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+First, weather is declared as a variable = undefined.
+Next, for weather = "sunny" (weather is defined with string "sunny").
+Then the final expression asks "is weather equal to sunny?" and
+it evalutes the boolean 'true'.
```

Comment:
Yep. Just keep in mind that the first two lines are expressions too, evaluating to `undefined` and `"sunny"`, respectively.